packages/react-native-windows-init/src/Cli.ts

```diff
@@ -368,12 +439,47 @@ function installReactNativeWindows(
   );
 }
 
+function isMSFTInternal(): boolean {
+  return (
+    process.env.USERDNSDOMAIN !== undefined &&
+    process.env.USERDNSDOMAIN.endsWith('microsoft.com')
```

Comment:
Same as other PR... Perhaps a '.' in front would make sense to catch full name...